docs/translations/translations.json

```
@@ -85,6 +85,9 @@
   "toggleRTL": "Toggle right-to-left/left-to-right",
   "toggleTheme": "Toggle light/dark theme",
   "traffic": "Traffic",
+  "v5IsOut": "v5.beta is out! 🎉 Head to the",
```

```suggestion
  "v5IsOut": "v5 beta is out! 🎉 Head to the",
```
The SemVer notation is irrelevant here.